README.md

```diff
@@ -465,6 +466,24 @@ Authenticating events by verifying signature
 | clientSecret | <code>string</code> | Client secret can be retrieved from the Adobe I/O Console integration |
 | signatureHeaderValue | <code>string</code> | Value of x-adobe-signature header in each POST request to the registered webhook URL |
 
+<a name="EventsCoreAPI+verifyDigitalSignatureForEvent"></a>
+
+### eventsCoreAPI.verifyDigitalSignatureForEvent(event, recipientClientId, signatureOptions) ⇒ <code>boolean</code>
```

Comment:
We also have a [types.d.ts](https://github.com/adobe/aio-lib-events/blob/master/types.d.ts) file. We should add the `verifyDigitalSignatureForEvent()` function and the `signatureOptions` object there as well.